base_edifact/models/edifact.py

```diff
@@ -200,19 +200,12 @@ def map2odoo_product(self, seg, pia=None):
         SA. Supplier's Article Number
         """
         code = seg[2][0]
-        product_tmp = self.env["product.template"]
         if code:
-            field = "default_code" if seg[2][1] == "SRV" else "barcode"
-            record = product_tmp.search([(field, "=", code)], limit=1)
```

Comment:
not doing these product search here does not have any impact later?

Reply:
No impact. Because later there will be a part to check whether the barcode exists or not.

Reply:
ok thanks, then ignore my other comments @QuocDuong1306 
maybe a small explanation might be wise in the final OCA pr description to avoid anybody thinking there is a regression here